prac_06/guitar.py

```diff
@@ -0,0 +1,28 @@
+"""Class defining a guitar."""
+
+CURRENT_YEAR = 2021     # Improvement: Use computer's internal clock
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Represents a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance."""
+        self.name = name
+        self.year = int(year)
+        self.cost = float(cost)
+
+    def __str__(self):
+        """String form of the instance."""
+        return "{} ({}) : ${:,.2f}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        """Get age of guitar."""
+        age = CURRENT_YEAR - self.year
```

Comment:
This function could be simplified into one line by writing it as: `return CURRENT_YEAR - self.year`

Reply:
Thanks.